cpp/src/ray/util/process_helper.h

```diff
@@ -37,11 +38,16 @@ class ProcessHelper {
     return processHelper;
   }
 
+  std::unique_ptr<ray::gcs::GlobalStateAccessor> CreateGlobalStateAccessor(
+      const std::string &redis_address, const std::string &redis_password);
+
   ProcessHelper(ProcessHelper const &) = delete;
   void operator=(ProcessHelper const &) = delete;
 
  private:
   ProcessHelper(){};
 };
+
+std::string GetNodeIpAddress(const std::string &address = "8.8.8.8:53");
```

Comment:
Can we build a new file named util.cc for `GetNodeIpAddress `? It's a little weird to expose this API in process helper.